core/cmd/remote_client.go

```diff
@@ -330,10 +331,11 @@ func (cli *Client) parseResponse(resp *http.Response) ([]byte, error) {
 	if errors.Is(err, errUnauthorized) {
 		return nil, cli.errorOut(multierr.Append(err, fmt.Errorf("your credentials may be missing, invalid or you may need to login first using the CLI via 'chainlink admin login'")))
 	}
+	if errors.Is(err, errForbidden) {
+		return nil, cli.errorOut(multierr.Append(err, fmt.Errorf("this action requires admin privileges current user does not have enough privileges to perform this action, login with admin via 'chainlink admin login'")))
```

Comment:
```suggestion
		return nil, cli.errorOut(multierr.Append(err, fmt.Errorf("this action requires admin privileges. The current user does not have enough privileges to perform this action, login with admin via 'chainlink admin login'")))
```

Reply:
Changed in [87d1067](https://github.com/smartcontractkit/chainlink/pull/8311/commits/87d1067a70e5b422e3763efba0f37954c7733777)